### ISSUE TITLE
Local temperature not recognized

### ISSUE THREAD

**Issue by timderspieler** [twhittock/avatto_me167#5]
Hello,

first off thank you so much for creating this integration! It worked flawlessly so far. But I have a small isssue with the local temperature. It seems like Zigbee2MQTT cannot pull any temperature data from the sensor. It will stay for me a 20°C.

This is the error I receive when trying to update the local temperature:

Zigbee2MQTT:error 2022-10-25 19:13:44: No converter available for 'local_temperature' ("")

Do you know how I can fix that?
My valve is branded by "onndo" and not Avatto. But it basically has the same product page (https://de.aliexpress.com/item/1005004696707285.html?spm=a2g0o.order_detail.0.0.9f7a6368PUP29U&gatewayAdapt=glo2deu) and is getting recognized just fine except the local temp :/ 

Can you please take a look at that?

Thanks in advance!

**Comment by twhittock** [comment]
Hi. Yeah, there doesn't seem to be a way to request information from the valve, that I can see. Local temperature is updated when it wants to, basically. Usually it will return an updated temperature when it notices a significant difference or some state on the TRV is changed (for example, if you change local temperature calibration, it will return the current temperature, taking account of the calibration.)

If you know of a way to either turn off the refresh buttons or actually get it to respond to a query then please let me know!

**Comment by timderspieler** [comment]
Thank you for the response!

I can actually confirm what you've said. The temperature changes once the trv recognizes a change (or if you change the offset). I first had to find out the actual offset. In my case it was 4!! degrees celcius. Now the temps seem to be pretty acurate once the trv refreshes them.

But yes an automatic refresh or a way to refresh it manually would be cool too. I'll let you know once I've found something.

PS: I hope your "script" will get into the final release of Z2M. Basically everybody that wants to use these cheap trvs will have the same problem as we had.
